### PR TITLE
Use `flake8` for fast lintint.

### PR DESCRIPTION
**Pull request by s0undt3ch**
PyLint should not be discarded since it does some deeper inspection, but flake8 is fast enough that we can even have it in a pre-commit hook.

### CODE REVIEW

File: tests/test_relocate.py [hunk 1]
```diff
@@ -226,9 +226,9 @@ def test_handle_elf(tmp_path):
 
     ldd_ret = """
     linux-vdso.so.1 => linux-vdso.so.1 (0x0123456789)
-	libcrypt.so.2 => {libcrypt} (0x0123456789)
-	libm.so.6 => /usr/lib/libm.so.6 (0x0123456789)
-	libc.so.6 => /usr/lib/libc.so.6 (0x0123456789)
+    libcrypt.so.2 => {libcrypt} (0x0123456789)
+    libm.so.6 => /usr/lib/libm.so.6 (0x0123456789)
+    libc.so.6 => /usr/lib/libc.so.6 (0x0123456789)
```
Comment (s0undt3ch) [inline tests/test_relocate.py:231]: Are the tabs really important here?

File: tests/test_relocate.py [hunk 2]
```diff
@@ -254,10 +254,10 @@ def test_handle_elf_rpath_only(tmp_path):
...
     ldd_ret = """
     linux-vdso.so.1 => linux-vdso.so.1 (0x0123456789)
-	libcrypt.so.2 => {libcrypt} (0x0123456789)
-	fake.so.2 => {fake} (0x0123456789)
-	libm.so.6 => /usr/lib/libm.so.6 (0x0123456789)
-	libc.so.6 => /usr/lib/libc.so.6 (0x0123456789)
+    libcrypt.so.2 => {libcrypt} (0x0123456789)
+    fake.so.2 => {fake} (0x0123456789)
+    libm.so.6 => /usr/lib/libm.so.6 (0x0123456789)
+    libc.so.6 => /usr/lib/libc.so.6 (0x0123456789)
```
Comment (s0undt3ch) [inline tests/test_relocate.py:260]: Are the tabs really important here?

File: setup.cfg [hunk 3]
```diff
@@ -21,3 +21,62 @@ include_package_data = True
 [options.entry_points]
 console_scripts =
     relenv = relenv.__main__:main
+
+[sdist]
+owner = root
+group = root
+
+[flake8]
```
Comment (dwoz) [inline setup.cfg:29]: Can this config reside in `pyproject.toml`?

File: tests/test_relocate.py (hunk 2 above)
Comment (dwoz) [reply tests/test_relocate.py:260]: By the looks of things, no.

**Review by dwoz — APPROVED** [review]

File: setup.cfg [hunk 4]
```diff
@@ -21,3 +21,62 @@ include_package_data = True
 [options.entry_points]
 console_scripts =
     relenv = relenv.__main__:main
+
+[sdist]
+owner = root
```
Comment (dwoz) [inline setup.cfg:26]: Is this definitely needed? https://github.com/pypa/setuptools/pull/2800

Comment (s0undt3ch) [reply setup.cfg:26]: yes, I believe `root` is not the default user. It'll use which ever user is creating the tarball.

File: setup.cfg (hunk 3 above)
Comment (s0undt3ch) [reply setup.cfg:29]: Apparently no.
https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations
https://github.com/PyCQA/flake8/blob/2680339377d9fdb0e27fc1f2f30828a93c60d08a/.github/ISSUE_TEMPLATE/02_feature.yml

**Pull request merged**